packages/plugins/src/plugins/silo/siloVaultsJob.ts

```diff
@@ -0,0 +1,133 @@
+import axios from 'axios';
+import { EvmNetworkIdType, formatEvmAddress } from '@sonarwatch/portfolio-core';
+import { Address } from 'viem';
+import { logger } from '@nx/devkit';
+import { Cache } from '../../Cache';
+import { Job, JobExecutor } from '../../Job';
+import {
+  PLATFORM_ID,
+  SILOS_GRAPH_ENDPOINT_1,
+  SILOS_GRAPH_ENDPOINT_2,
+  SILOS_GRAPH_ENDPOINT_3,
+  SILOS_VAULTS_KEY,
+  SILOS_REWARD_VAULTS_KEY,
+} from './constants';
+import { SiloVaultsResponse } from './types';
+
+const query = {
+  query: `{
```

Comment:
```suggestion
  query: `gql{
```
I think if you add this to the query it will give you automatic formating?